library/core/src/ptr/metadata.rs

```diff
@@ -180,10 +180,20 @@ pub struct DynMetadata<Dyn: ?Sized> {
     phantom: crate::marker::PhantomData<Dyn>,
 }
 
+/// Opaque type for accessing vtables.
+///
+/// Private implementation detail of `DynMetadata::size_of` etc.
+/// Must be zero-sized since there is conceptually not actually any Abstract Machine memory behind this pointer.
+/// However, we can require pointer alignment.
+#[repr(C)]
+#[cfg(not(bootstrap))]
+struct VTable([usize; 0]);
+
 /// The common prefix of all vtables. It is followed by function pointers for trait methods.
 ///
 /// Private implementation detail of `DynMetadata::size_of` etc.
 #[repr(C)]
+#[cfg(bootstrap)]
 struct VTable {
     drop_in_place: fn(*mut ()),
```

Comment:
was this field never read anywhere?

Reply:
It doesn't seem so.

Reply:
Yeah I have found no use of it. CI would tell.^^